Resources/Prototypes/Entities/Objects/Materials/shards.yml

```diff
@@ -1,18 +1,18 @@
 ﻿- type: entity
+  abstract: true
+  parent: BaseItem
   id: ShardBase
   name: shard
   description: It's a shard of some unknown material.
```

Comment:
Might want to remove this abstract name + description if you're going with that for the other ones.